### DISCUSSION TITLE
痛

### DISCUSSION THREAD

**Discussion by penzi114** [share121/inter-knot#2683] [其他]
每次进来都要挂梯子，一挂上网络质量就下降了…在手机上搞这些插件真的好痛苦啊，是只有我这样吗

**Comment by alanyagger** [comment]
为什么网络质量会下降 是没有设置路由分流策略吗还是梯子质量太差？我在PC上搭的vps一般是绕过国内域名

**Reply by dnmdngcsb** [reply]
对啊，可以选规则代理，不用挂全局啊，反正我PC上的梯子基本上天天挂着，也没什么影响啊

**Comment by penzi114** [comment]
我是移动端(T~T)用的set up vpn，每次进去真的超卡…



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月18日 15:09 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 痛 (Discussion #2683) |

为什么网络质量会下降 是没有设置路由分流策略吗还是梯子质量太差？我在PC上搭的vps一般是绕过国内域名

**Reply by dnmdngcsb** [reply]
……clash meta不好吗

**Comment by Maple013** [comment]
移动端没办法了。我这里PC用WT+Edge还是很舒适的。

**Comment by penzi114** [comment]
这样吗，谢谢指点，我去看看



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月18日 20:55 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 痛 (Discussion #2683) |

……clash meta不好吗
